src/utils/findpackage/packagewin.rs

```diff
@@ -9,103 +9,85 @@ use std::{
 use crate::Url;
 
 use super::{
-    get_version, handle_config_package, CMAKECONFIG, CMAKECONFIGVERSION, CMAKEREGEX,
-    SPECIAL_PACKAGE_PATTERN,
+    get_available_libs, get_cmake_message, get_version, handle_config_package, CMAKECONFIG,
+    CMAKECONFIGVERSION, CMAKEREGEX, SPECIAL_PACKAGE_PATTERN,
 };
 
-const LIBS: [&str; 4] = ["lib", "lib32", "lib64", "share"];
+pub(super) const LIBS: [&str; 4] = ["lib", "lib32", "lib64", "share"];
 
 pub static CMAKE_PACKAGES: LazyLock<Vec<CMakePackage>> =
     LazyLock::new(|| get_cmake_message().into_values().collect());
 
 pub static CMAKE_PACKAGES_WITHKEY: LazyLock<HashMap<String, CMakePackage>> =
     LazyLock::new(get_cmake_message);
 
-fn get_prefix() -> Option<String> {
-    if let Ok(mystem_prefix) = std::env::var("MSYSTEM_PREFIX") {
-        return Some(mystem_prefix);
+pub(super) fn get_env_prefix() -> Option<String> {
+    if let Ok(prefix) = std::env::var("MSYSTEM_PREFIX") {
+        return Some(prefix);
     }
     std::env::var("CMAKE_PREFIX_PATH").ok()
 }
 
-fn get_available_libs(prefix: &str) -> Vec<PathBuf> {
-    let mut ava: Vec<PathBuf> = Vec::new();
-    let root_prefix = Path::new(&prefix);
-    for lib in LIBS {
-        let p = root_prefix.join(lib).join("cmake");
-        if p.exists() {
-            ava.push(p);
-        }
-    }
-    ava
-}
-
 fn safe_canonicalize<P: AsRef<Path>>(path: P) -> std::io::Result<PathBuf> {
-    use path_absolutize::Absolutize;
     Ok(path.as_ref().absolutize()?.into_owned())
```

Comment:
here this line (path.as_ref().absolute())

the function absolute if from the `path_absolute::Absolutize`